### PR TITLE
update analytic rules of ZINC actor

### PR DESCRIPTION
**Pull request by manishkumar1991**
Required items, please complete
   
   Change(s):
   - Updated with new mapping and minor logic fixes

   Reason for Change(s):
   - Old Rules were not updated 

   Version Updated:
   - N/A

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/Zinc Open Source/Analytic Rules/ZincOctober2022_IP_Domain_Hash_IOC.yaml [hunk 1]
```diff
@@ -131,52 +130,59 @@ query:  |
...
   | project TimeGenerated, UserAgent, Operation, RecordType, UserId, ClientIP, Type
-  | extend timestamp = TimeGenerated, IPCustomEntity = ClientIP, AccountCustomEntity = UserId
+  | extend timestamp = TimeGenerated, IPEntity = ClientIP, AccountEntity = UserId
   )
   )
+  | extend HostName = tostring(split(HostEntity, '.', 0)[0]), DnsDomain = tostring(strcat_array(array_slice(split(HostEntity, '.'), 1, -1), '.'))
 entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
```
Comment (rahul0216) [inline Solutions/Zinc Open Source/Analytic Rules/ZincOctober2022_IP_Domain_Hash_IOC.yaml:161]: Use Name and UPNSuffix identifier.

**Review by rahul0216 — COMMENTED** [review]
Please address the inline comment.

File: Solutions/Zinc Open Source/Analytic Rules/ZincOctober2022_IP_Domain_Hash_IOC.yaml (hunk 1 above)
Comment (manishkumar1991) [reply Solutions/Zinc Open Source/Analytic Rules/ZincOctober2022_IP_Domain_Hash_IOC.yaml]: Done

**Review by rahul0216 — APPROVED** [review]
Updates look good.

**Pull request merged**